### PR TITLE
Fix: Remove Unnecessary Comma in Private Forks Documentation

### LINKED ISSUE #22494: Remove unnecessary comma in private forks test
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#enabling-workflows-for-forks-of-private-repositories

### What part(s) of the article would you like to see updated?

> If you rely on using forks of your private repositories, you can configure policies that control how users can run workflows on pull_request events. Available to private repositories only, you can configure these policy settings for organizations, or repositories.

If you can set this knob for enterprises, the text should say so.

If you can't set it for enterprises, the text should say so.

### Additional information

#22493

### PR DESCRIPTION
**Pull request by Sriparno08**
### Why:

There is an unnecessary comma in the `private-repository-forks-overview.md` document.

Version: fpt (Free, Pro, or Team plans)

Closes #22494

### What's being changed (if available, include any code snippets, screenshots, or gifs):

I have removed the unnecessary comma in the `private-repository-forks-overview.md` document.

GitHub URL: https://github.com/github/docs/blob/main/data/reusables/actions/private-repository-forks-overview.md#L0-L1

GitHub Docs URL: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@Sriparno08 Thanks for submitting a PR! Per this [comment](https://github.com/github/docs/issues/22494#issuecomment-1345968639), the comma is still necessary for certain versions of documentation and needs to be versioned for those instances. Can you update the PR to include the necessary versioning? You can find out more about our Liquid helpers for versioning [here](https://github.com/github/docs/blob/main/contributing/liquid-helpers.md).

**Comment by Sriparno08** [comment]
Updated it, @cmwilson21!

File: data/reusables/actions/private-repository-forks-overview.md [hunk 1]
```diff
@@ -1 +1 @@
-If you rely on using forks of your private repositories, you can configure policies that control how users can run workflows on `pull_request` events. Available to private {% ifversion ghec or ghes or ghae %}and internal{% endif %} repositories only, you can configure these policy settings for {% ifversion ghec %}enterprises, {% elsif ghes or ghae %}your enterprise, {% endif %}organizations, or repositories.
+If you rely on using forks of your private repositories, you can configure policies that control how users can run workflows on `pull_request` events. Available to private {% ifversion ghec or ghes or ghae %}and internal{% endif %} repositories only, you can configure these policy settings for {% ifversion ghec %}enterprises, {% elsif ghes or ghae %}your enterprise, {% endif %}organizations or repositories.
```
Comment (lecoursen) [inline data/reusables/actions/private-repository-forks-overview.md:1]: When versioning gets too complicated, it's okay to allow a little bit of repetition in order to make the source easier to read. 😊 
```suggestion
If you rely on using forks of your private repositories, you can configure policies that control how users can run workflows on `pull_request` events. {% ifversion fpt %}Available to private repositories only, you can configure these policy settings for organizations or repositories.{% else %}Available to private and internal repositories only, you can configure these policy settings for {% ifversion ghec %}enterprises{% else %}your enterprise{% endif %}, organizations, or repositories.{% endif %}
```

**Comment by Sriparno08** [comment]
Correctly said, @lecoursen 👍

**Review by lecoursen — APPROVED** [review]

**Review by nitindudeja2107 — COMMENTED** [review]
#22914

**Pull request merged**